independent-publisher-connectors/Secure Code Warrior/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7047 @@
+{

```

Comment:
@troystaylor Please remove artifacts related to other connectors from this PR.

Reply:
Done